src/Sylius/Bundle/AdminBundle/tests/Notification/HubNotificationProviderTest.php

```
@@ -80,72 +76,179 @@ public function setUp(): void
-        $request->withHeader('Content-Type', 'application/json')->willReturn($request);
-        $request->withBody($stream)->willReturn($request);
-
-        $this->client->sendRequest(Argument::cetera())->willThrow(ConnectException::class);
+        $request = $this->createMock(RequestInterface::class);
+        $stream = $this->createMock(StreamInterface::class);
+
+        $this->cache
+            ->expects($this->once())
+            ->method('get')
+            ->with('latest_sylius_version', $this->isType('callable'))
+            ->willReturnCallback(fn ($key, $callback) => $callback());
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)